src/actions/MapActions.js

```diff
@@ -23,8 +23,7 @@ export const initializeMap = ({region}) => {
         });
 
         firebase.firestore().collection('campsites')
-            .get()
-            .then(querySnapshot => {
+            .onSnapshot(querySnapshot => {
```

Comment:
Awesome little trick with Firebase. Instead of making a static `get` call, if we do `onSnapshot`, it creates a listener which waits for any changes from the firebase list and automatically re-triggers this action any time a new site is added. This makes our logic of having to fire off specific actions when we add a new site MUCH cleaner. Now all we do is just add the site in the `AddSiteActions`, and then when firebase gets the info, it'll fire this action again, and we get a fully updated list of sites.